src/plugins/graphql.js

```diff
@@ -55,12 +55,20 @@ function createWrapParse (tracer, config) {
         const operation = getOperation(document)
 
         if (!operation) return document // skip schema parsing
+        let operationValue = ''
+        operationValue = `${operation.operation}`
+        if (operation.name) {
+          operationValue = `.${operationValue}.${operation.name.value}`
+        } else {
+          operationValue = `.${operationValue}.${operation.selectionSet.selections[0].name.value}`
```

Comment:
How certain is the existence of these properties?  Can there be some safety checks?